apiml-static-reg.yaml.template

```diff
@@ -0,0 +1,21 @@
+services:
+  - serviceId: zss
+    title: Zowe System Services (ZSS)
+    description: 'TEST Zowe System Services is an HTTPS and Websocket server that makes it easy to have secure, powerful web APIs backed by low-level z/OS constructs. It contains services for essential z/OS abilities such as working with files, datasets, and ESMs, but is also extensible by REST and Websocket "Dataservices" which are optionally present in App Framework "Plugins".'
+    catalogUiTileId: zss
+    instanceBaseUrls:
+      - ${PROTOCOL}://${ZOWE_EXPLORER_HOST}:${ZOWE_ZSS_SERVER_PORT}/
```

Comment:
Please replace `${PROTOCOL}` by `https`. `https` is a default now for zss. In anyway, when zss was running on `http://127.0.0.1:port` it was never accessible through APML Gateway.